### ISSUE TITLE
Ctrl + D Keyboard shortcut

### ISSUE THREAD

**Issue by FMalerba** [files-community/Files#5673]
### Solution/Idea

```markdown
Traditional Windows' File Explorer comes with the ctrl+D keyboard shortcut to delete selected files and folders instead of having to click around with your mouse. This is a very much need feature in my opinion.
```

### Alternatives

```markdown
Currently available alternatives all seem to require mouse usage, which may be undesirable sometimes.
```

### Priorities

```markdown
| Capability | Priority |
| :---------- | :------- |
| This proposal will allow developers to accomplish W | Must |
| This proposal will allow end users to accomplish X | Should |
| This proposal will allow developers to accomplish Y | Could |
| This proposal will allow end users to accomplish Z | Won't |
```

### Files Version

Versione 1.9.18.0 

### Windows Version

Windows 11 Home 21H2 - 22000.100

**Comment by gave92** [comment]
Thanks for the suggestion. The supported keyboard shortcut is Delete key, Cltrl + D can still be added.

**Comment by yaichenbaum** [comment]
This issue should be resolved with #5674.
